Formula/emscripten.rb

```diff
@@ -60,16 +62,32 @@ def install
     (buildpath/"fastcomp").install resource("fastcomp")
     (buildpath/"fastcomp/tools/clang").install resource("fastcomp-clang")
 
-    args = [
+    configure_args = [
       "--prefix=#{libexec}/llvm",
       "--enable-optimized",
       "--enable-targets=host,js",
       "--disable-assertions",
       "--disable-bindings",
     ]
 
+    cmake_args = [
+      "-DCMAKE_INSTALL_PREFIX=#{libexec}/llvm",
+      "-DCMAKE_BUILD_TYPE=Release",
+      "-DLLVM_TARGETS_TO_BUILD=X86;JSBackend",
+      "-DLLVM_INCLUDE_EXAMPLES=OFF",
+      "-DLLVM_INCLUDE_TESTS=OFF",
+      "-DCLANG_INCLUDE_EXAMPLES=OFF",
+      "-DCLANG_INCLUDE_TESTS=OFF",
+      "-DOCAMLFIND=/usr/bin/false",
+      "-DGO_EXECUTABLE=/usr/bin/false",
+    ]
```

Comment:
Before inlining, I noticed `std_cmake_args` is defined for CMake builds in Homebrew.
I checked it and found that can be applicable for this fastcomp compilation.
Can I include this standard argument (and thus persist on the array) ?